### PR TITLE
Adding integ test for serverlessrepo 1.10.0

### PR DESCRIPTION
**Pull request by c2tarun**
*Issue #, if available:*
Integration test for https://github.com/awslabs/aws-sam-cli/pull/1344
*Why is this change necessary?*

*How does it address the issue?*

*What side effects does this change have?*

*Did you change a dependency in `requirements/base.txt`?*
    *If so, did you run `make update-reproducible-reqs`*

*Checklist:*

- [ ] Write Design Document ([Do I need to write a design document?](https://github.com/awslabs/aws-sam-cli/blob/develop/DEVELOPMENT_GUIDE.rst#design-document))
- [ ] Write unit tests
- [ ] Write/update functional tests
- [ ] Write/update integration tests
- [x] `make pr` passes
- [ ] Write documentation

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: tests/integration/publish/test_command_integ.py [hunk 1]
```diff
@@ -32,67 +37,65 @@ def setUp(self):
     def tearDown(self):
         super(TestPublishExistingApp, self).tearDown()
         # Delete application for each test
-        self.sar_client.delete_application(ApplicationId=self.application_id)
+        # self.sar_client.delete_application(ApplicationId=self.application_id)
```
Comment (sriram-mv) [inline tests/integration/publish/test_command_integ.py:40]: should this be uncommented?

Comment (c2tarun) [reply tests/integration/publish/test_command_integ.py:40]: Yes, I added that to debug, I'll fix this.

**Comment by jfuss** [comment]
Please run `make update-reproducible-reqs` so the pinned versions are correctly updated.

File: tests/integration/publish/test_command_integ.py [hunk 2]
```diff
@@ -1,19 +1,24 @@
...
 from .publish_app_integ_base import PublishAppIntegBase
 from tests.testing_utils import RUNNING_ON_CI, RUNNING_TEST_FOR_MASTER_ON_CI, RUN_BY_CANARY
+from tests.testing_utils import run_command
 
 # Publish tests require credentials and CI/CD will only add credentials to the env if the PR is from the same repo.
 # This is to restrict publish tests to run outside of CI/CD, when the branch is not master and tests are not run by Canary.
 SKIP_PUBLISH_TESTS = RUNNING_ON_CI and RUNNING_TEST_FOR_MASTER_ON_CI and not RUN_BY_CANARY
 TIMEOUT = 300
 
+LOG = logging.getLogger(__name__)
```
Comment (jfuss) [inline tests/integration/publish/test_command_integ.py:20]: Looks unused? If so, please remove (and the `import logging`).

**Comment by aahung** [comment]
Close, integration tests were added in https://github.com/aws/aws-sam-cli/pull/2305

**Pull request closed without merging**